tensorflow/core/kernels/mkl/mkl_conv_ops.cc

```diff
@@ -1521,6 +1521,15 @@ class MklFusedConvOp
           context, num_args == 4,
           errors::InvalidArgument(
               "Fused Conv2D with batchnorm must have 4 extra argument"));
+      std::vector<DataType> TArgs;
+      OP_REQUIRES_OK(context, context->GetAttr("TArgs", &TArgs));
+      OP_REQUIRES(
+          context, TArgs.size() == 4,
+          errors::InvalidArgument("FusedConv2D TArgs size expected to be 4"));
```

Comment:
Please replace `errors::InvalidArgument` with `absl::InvalidArgumentError`. Same for other kinds of `errors::*`. You can search for example usage in the repo, e.g.,
https://github.com/tensorflow/tensorflow/blob/b7f213c61d79dfaa359b0e0de22e773e2fcb538c/tensorflow/cc/framework/ops.h#L161-L163

Please do this for all files in this PR and in future PRs.

Reply:
@penpornk  - I changed all occurances of errors::* in mkl_conv_ops.cc. Also, needed to use absl::StrCat() and adjust formatting due to which you might see more LOC. No errors::* used in any other files in this PR.